modules/src/ics24_host/path.rs

```diff
@@ -11,7 +11,7 @@ pub const IBC_QUERY_PATH: &str = "store/ibc/key";
 pub enum Path {
     ClientType(ClientId),
     ClientState(ClientId),
```

Comment:
Could you change the client state to be provable in `is_provable()` below? This was changed in the last few versions (protobuf, sdk, etc).

Reply:
Changed to true. I was not aware of this change, good to know!